services-manager/README.md

```diff
@@ -0,0 +1,103 @@
+<!---
+Copyright 2019 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# <img src="../logos/logo.svg" width="180">
+
+### Palisade Services Manager
+
+
+## Documentation
+
+The documentation for the latest release can be found [here](https://gchq.github.io/Palisade).
+
+
+## Getting started
+
+The Palisade Services Manager is a configuration-driven process-spawner and REST-client with particular considerations to assist in managing multiple Palisade services running on local JVMs.
+
+The manager is designed to be used by defining a collection of SpringBoot configuration files:
+ * One set for the manager itself - this defines how it should start up further services or reconfigure those already running
+ * One set for each service - these are discovered through the above configuration and passed on as an argument to the service jar
+    * These wll all already be located in each service's `resources` directory
+ 
+ The manager works in several modes, operated by the use of several flags:
+ * `--run` - run the configured services, spawning a new process for each entry
+ * `--logging` - perform a change to the logging level of configured services by REST requests
+ * `--config` - print out a human-readable view of the spring-boot configuration given to the services-manager
+ 
+ 
+## Examples
+
+### Starting Services
+If services are already running, using the built-in profiles:  
+ 1. Start Eureka - run `java -jar -Dspring.profiles.active=eureka services-manager-exec.jar --run` and wait for Eureka to start up on `localhost:8083`
```

Comment:
there is no jar called services-manager-exec.jar. put in the path from the project root and use a maven plugin to embed the current build jar name into the readme. People need to be able to just copy-pate these into the window.

Reply:
Resolved with maven-resources-plugin/copy-resources and maven-replacer-plugin/replace